### PR TITLE
Add tip about making dropdown height fit 5 langs.

### PR DESCRIPTION
**Pull request by agustif**
#### Description of what you did:

Just added a tip on how to enlarge localeDropdown height's to fit 5 languages.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [x] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by lauriejim** [comment]
Hello! Thank you for this PR but I'm not sure it's necessary to add this into the docs.
@soupette can you confirm please.

**Comment by agustif** [comment]
> Hello! Thank you for this PR but I'm not sure it's necessary to add this into the docs.
> @soupette can you confirm please.

Yeah prob not necessary, it was an easy fix and anyone can come to the conclusion itself.

But I would say I did have 5 minuts of -doubts- when adding the fifth language in my admin-ui, it wouldn't show up in the dropdown because of the height.

Since I have osx+touchpad, I don't see scrollbars until I scroll 'em

Maybe there's a better place for this, or it's just too of a minor case that it will be a problem for more than 1% of the userbase, Idk.

Maybe you've some telemetry going on and know which percentage of users use 4 langs or more for the admin.

Anyway will be OK with me, I understand this is not important and low priority and might not make it to master, maybe it's helpful as a closed pull request someone can reference if searching for it.

File: docs/3.0.0-beta.x/admin-panel/customization.md [hunk 1]
```diff
@@ -127,6 +127,15 @@ export { languages, translationMessages };
 With this modification only English and French will be available in your admin
 :::
 
+::: tip
+If you need 4+ languages to show in the dropdown you can change the css by copying the file from cache and customizing the css yourself 
+:::
+cp .cache/admin/src/containers/LocaleToggle/Wrapper.js admin/src/containers/LocaleToggle/Wrapper.js
+edit Wrapper.js:67 "max-height: 192px !important;"
```
Comment (soupette) [inline docs/3.0.0-beta.x/admin-panel/customization.md]: Hi thanks for adding this to the documentation but I don't really see the difference with the `Wrapper.js` override...

Comment (agustif) [reply docs/3.0.0-beta.x/admin-panel/customization.md]: Yeah np, prob no need to add it to the docs

**Comment by lauriejim** [comment]
OKay, we will not add this in the docs.
Thank you.

**Pull request closed without merging**